test/fixtures/blueprint-data.coffee

```diff
@@ -0,0 +1,141 @@
+module.export = {
+  "./path/to/blueprint": {
```

Comment:
I'd prefer to see it with some extension, thus anyone else wouldn't be mistaken it is path to directory, but path to a file.
-> `"./path/to/blueprint.file"`